.github/workflows/pkgci_regression_test.yml

```
@@ -208,6 +208,19 @@ jobs:
         env:
           ROCM_CHIP: ${{ matrix.rocm-chip }}
 
+      - name: "Running llama special model tests"
+        if: "!cancelled()"
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest ./experimental/regression_suite/shark-test-suite-models/llama \
+            -k ${{ matrix.backend }} \
+            -rpfE \
+            --capture=no \
+            --log-cli-level=info \
+            --timeout=240 \
+            --durations=0
+        env:
+          ROCM_CHIP: ${{ matrix.rocm-chip }}
       # Note: mi250 benchmark times are more lenient than mi300 (allowing about
```

Please add a newline between groups in this file
```suggestion
          ROCM_CHIP: ${{ matrix.rocm-chip }}

      # Note: mi250 benchmark times are more lenient than mi300 (allowing about
```